src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -274,12 +274,12 @@ public void CreateWalletWithMnemonicListCreatesWalletUsingMnemonicList()
 
             string password = "test";
 
-            var mnemonicList = new Mnemonic(Wordlist.French, WordCount.Eighteen);
+            var mnemonic = new Mnemonic(Wordlist.French, WordCount.Eighteen);
 
             // create the wallet
-            Mnemonic mnemonic = walletManager.CreateWallet(password, "mywallet", mnemonicList: mnemonicList.ToString());
+            var returnedMnemonic = walletManager.CreateWallet(password, "mywallet", null, mnemonic);
```

Comment:
Can you please change var to Mnemonic here? 
It's already not very intuitive that a CreateWallet method shall return a mnemonic.